modules/usage/module.php

```diff
@@ -564,22 +574,50 @@ private function save_document_usage( Document $document ) {
 		// Get data manually to avoid conflict with `\Elementor\Core\Base\Document::get_elements_data... convert_to_elementor`.
 		$data = $document->get_json_meta( '_elementor_data' );
 
-		if ( ! empty( $data ) ) {
-			try {
-				$usage = $this->get_elements_usage( $document->get_elements_raw_data( $data ) );
+		try {
+			$this->add_document_usage( $document, $data );
+		} catch ( \Exception $exception ) {
+			return; // Do nothing.
+		};
+	}
+
+	private function add_document_usage( $document, $data ) {
+		if ( $this->should_skip_document( $document ) ) {
+			return;
+		}
```

Comment:
Remove. get the usage from the document itself.

Reply:
Handled